### PR TITLE
Setup react app

### PR DESCRIPTION
**Pull request by yonas44**
For this PR, I included:
- Create react app using CRA.
- Setup linters for react.

Thank you for reviewing my PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
-# math_magicians
\ No newline at end of file
+A todo-list project built with Webpack and ES6 modules
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```
Comment (Olamarx) [inline README.md]: - [ ] Kindly use the latest readme file. You can find it [here](https://github.com/microverseinc/readme-template)

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @yonas44, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✅
- CRA ✅ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (yonas44) [reply README.md]: Okay thanks for the feedback, I will address that.

**Review by Olamarx — APPROVED** [review]
Hi @yonas44,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✅
- CRA ✅
- Readme file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**